doc/source/conf.py

```diff
@@ -18,6 +18,10 @@
 from importlib.metadata import version
 from pathlib import Path
 
+autodoc_mock_imports = [
+    "diffpy.utils.wx",
```

Comment:
what was this added?

Reply:
sphinx gave an error `WARNING: autodoc: failed to import module 'gridutils' from module 'diffpy.utils.wx'; the following exception was raised:
No module named 'wx' [autodoc.import_object]`
So I followed the cookiecutter instructions for that error